src/model_signing/signing/in_toto.py

```diff
@@ -317,10 +204,10 @@ def from_manifest(cls, manifest: manifest_module.Manifest) -> Self:
             An instance of `DigestOfDigestsIntotoPayload`.
 
         Raises:
-            TypeError: If the manifest is not `FileLevelManifest`.
+            TypeError: If the manifest is not `Manifest`.
         """
-        if not isinstance(manifest, manifest_module.FileLevelManifest):
-            raise TypeError("Only FileLevelManifest is supported")
+        if not isinstance(manifest, _manifest.Manifest):
+            raise TypeError("Only Manifest is supported")
```

Comment:
are we losing any guarantees here (and below) in terms of distinguishing file-level and shard-level?